test/unit/tab/index.test.tsx

```diff
@@ -29,6 +29,16 @@ test('adds the active class if props.active is true on mount', () => {
   assert.isTrue(wrapper.hasClass('mdc-tab--active'));
 });
 
+test('sets the tabIndex to 0 if props.active is true on mount', () => {
+  const wrapper = shallow(<Tab active />);
+  assert.equal(wrapper.prop('tabIndex'), 0);
+});
+
+test('sets the tabIndex to -1 if props.active is false on mount', () => {
+  const wrapper = shallow(<Tab active={false} />);
```

Comment:
Could we update this to set no active property instead of explicitly setting `active={false}`? Will the test still pass?

Reply:
Typically with these props, it wouldn't be hard-coded. It would usually look more like:
```
<Tab active={this.state.active} />
```

In that case it is more helpful to have that prop explicitly defined like that. Otherwise removing it would look like:

```
const props = {};
if (this.state.active) {
  props.active = true;
}
<Tab {...props} />
```

Reply:
Fair enough, thanks for explaining. My thinking was that removing the `active={false}` prop tests that a `Tab` has `tab-index=-1` by default without the user providing any active prop. However, that's more a test of `Component.defaultProps` working than of the change here.